kivy_ios/recipes/materialyoucolor/__init__.py

```diff
@@ -0,0 +1,17 @@
+from kivy_ios.toolchain import PythonRecipe
+
+
+class MaterialYouColorRecipe(PythonRecipe):
+    version = "2.0.7"
+    url = "https://github.com/T-Dynamos/materialyoucolor-pyhton/archive/refs/tags/v{version}.tar.gz"
+    depends = ["hostpython3", "python3"]
```

Comment:
```suggestion
    depends = ["hostpython3", "python3"]
    hostpython_prerequisites = ["certifi"]
```

Reply:
@T-Dynamos , FYI: this one should fix the build, as you should ensure `certifi` is installed on the `hostpython3` environment.